modules/module_service.go

```diff
@@ -27,6 +27,14 @@ type moduleService struct {
 	startDeps, stopDeps func(string) map[string]services.Service
 }
 
+func (w moduleService) ServiceName() string {
+	msg := fmt.Sprintf("module service %s", w.name)
```

Comment:
I think it's "service" for "module XXX". Not sure I would understand "module service XXX" as having the same meaning.

Reply:
yeah, `service` and `module` aren't very clear terms anyways. 

should we just drop `service`? the struct is an implementation of `Service`. None of the other services mention that they are a service.

I feel like we're spending time without too much added benefit, if we can't reach consensus, I'm happy to close this PR

Reply:
so I added a decorator that adds the `NamedService` implementation. The decorator is used when the wrapped service already implements `NamedService`. I changed the constructor function of `moduleService`. Doesn't look too bad.


on another note I noticed the `services.DescribeService` function which does kinda what I needed in the first place - a way to describe a service.

https://github.com/grafana/dskit/blob/0e4b5def51c060fc498de65ee5b0a20563891159/services/services.go#L135-L145